src/docs/asciidoc/hazelcast_overview.adoc

```diff
@@ -293,6 +176,32 @@ In these cases, the partition table in the oldest member is updated with the new
 ownerships. Note that if a lite member joins or leaves a cluster, repartitioning is not triggered
 since lite members do not own any partitions.
 
+[[use-cases]]
+=== Use Cases
+
+Hazelcast can be used:
+
+* to share server configuration/information to see how a cluster performs
```

Comment:
I really don't understand this first point. Do you?

Reply:
Leftover from the last review, removing.